### ISSUE TITLE
Can't find mount path  in /etc/fstab

### ISSUE THREAD

**Issue by BartokIT** [ansible-collections/ansible.posix#180]
##### SUMMARY
I've used the mount module on several hosts without problem but fails on some hosts where there isn't  newline at the end of the fstab file.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
ansible.posix.mount

##### ANSIBLE VERSION
2.10

##### CONFIGURATION
```paste below
ANSIBLE_FORCE_COLOR(env: ANSIBLE_FORCE_COLOR) = True
```

##### OS / ENVIRONMENT
Red Hat Enterprise Linux Server release 7.8 (Maipo)

##### STEPS TO REPRODUCE
The playbook simply mount an already created path and file system.

```yaml
- name: Mount the created filesystems and add them to the fstab
  mount:
    path: /opt/ibm/itm     
    src: /dev/mapper/rootvg-lv_opt_ibm_itm     
    fstype: xfs
    opts: defaults
    state: mounted


```

##### EXPECTED RESULTS
The mounting process go fine

##### ACTUAL RESULTS
The module fails with the following error messages:

```paste below
failed: [c2tsttms6] 
{"msg": "Error mounting /opt/ibm/itm: mount: /etc/fstab: parse error: ignore entry at line 22.\nmount: can't find /opt/ibm/itm  in /etc/fstab\n"} 
```
Adding a newline at the end of the /etc/fstab  solve the problem.

**Comment by saito-hideki** [comment]
Hi @BartokIT 
I could not replicated this issue on my testing environments like below. There is no newline in the /etc/fstab but mount module is able to mount and umount /mnt:
```
# cat /etc/fstab
...snip...
/dev/mapper/rhel-swap   swap                    swap    defaults        0 0
/dev/vdb1		/mnt			xfs	defaults	0 0[root@server ~]#
# hexdump -c /etc/fstab
...snip...
00001d0  \n   /   d   e   v   /   v   d   b   1  \t  \t   /   m   n   t
00001e0  \t  \t  \t   x   f   s  \t   d   e   f   a   u   l   t   s  \t
00001f0   0       0```
```
[Environment]
- Control Note: CentOS Stream release 8
- Managed Note(Target): Red Hat Enterprise Linux Server release 7.9 (Maipo)
- Ansible version: 
  - 2.9.22
  - 2.10.0(ansible-base 2.10.0)
  - 3.0.0(ansible-base 2.10.10)
  - 4.0.0(ansible-core 2.11.1)
  - 2.12.0.dev0 with latest ansible.posix collection(2021-06-21)

[Test playbook task]
```
    - ansible.posix.mount:
        src: /dev/vdb1
        path: /mnt
        opts: defaults
        state: mounted
        fstype: xfs
        state: mounted
      become: true

    - ansible.posix.mount:
        src: /dev/vdb1
        path: /mnt
        state: unmounted
      become: true
```

If you are still running into the issue, can you provide the details a little more?  I'm not a maintainer of this module but it will help maintainers for further investigation:
- Ansible version within revision number like 2.10.X
- The failed task output of your playbook with -vvv option
- /etc/fstab file which you are seeing the issue
- /etc/fstab file which after added newline

**Comment by saito-hideki** [comment]
@BartokIT I think this issue has been fixed in #211. So I'm closing this, but if you still have the same issue with the latest version, please feel free to reopen this issue.
